README.md

```diff
@@ -1 +1,73 @@
-# Mobile-version
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```

Comment:
- [  ] please as the previous code reviewer said customize your README to fit your information because it is very important to display a summary about the project💪🏻